Builds/Test.py

```diff
@@ -129,14 +169,76 @@ def powerset(iterable):
     help='Reduce output where possible (unit tests)',
 )
 
+# Scons and CMake parameters are too different to run
+# both side-by-side
+pgroup = parser.add_mutually_exclusive_group()
+
+pgroup.add_argument(
+    '--cmake', '-m',
```

Comment:
`-m`? This command and some others below have some quite unintuitive one letter abbreviations...